internal/api.go

```diff
@@ -43,6 +37,38 @@ func NewAPI(config api.Config) (api.API, error) {
 	}, nil
 }
 
+func loadRules(conversionRulesPath string) (RuleSet, error) {
+	ruleSet := RuleSet{
+		rules: []Rule{},
+	}
+
+	filenames, err := filepath.Glob(filepath.Join(conversionRulesPath, "*.yaml"))
+	if err != nil {
+		return RuleSet{}, err
+	}
+
+	for _, filename := range filenames {
+		file, err := os.Open(filename)
```

Comment:
The Go idiom is to have `defer file.Close();` immediately after opening a file (unless it needs to stay open after you return for some reason).